src/controllers/api/clients/courses.js

```diff
@@ -0,0 +1,35 @@
+import { getToken } from '../../create_token';
+import models from '../../../database/models';
+
+
+export const postCourse = (req, res) => {
+  const courseId = req.params.id;
+  const { token } = req.cookies;
+  getToken(token,(getTokenErr, data) => {
```

Comment:
this what i'm talking about 
no need to do this every thime

Reply:
I am not really sure,
as i said before this is a different request than the one who has the data on it